uint32 sample_id
string sample_name
float32 value